{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/{{cookiecutter.app_name}}/info_views.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
Do we want to create default urls for these? Or is that up to the project developers?

Reply:
Good shout!